must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -191,20 +191,36 @@ for ns in $namespaces; do
         done
     done
 
-    # Add Ready nodes to the list
-    nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
+    #Add ocs labelled nodes to the list
+    nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers|awk '{print $1}')
```

Comment:
Is this something we want? Do we want to try and run debug Pods on nodes that aren't ready? @crombus does this make sense? I suppose worst case might be that we have to wait the full five minutes for the timeout to be hit, and it happens in parallel... still, seems odd.

Reply:
I will revert the changes back to the initial state for this one.

Reply:
Updated!.